avro-schema/batch-signature.avsc

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I suppose there isn't a lot of structure to the batch header signature. Is it simpler to have it as a raw signature file?

Reply:
That's a good idea and would save us some serialization ceremony. We could just store the signature bytes in the .sig files.

Reply:
Since we're now only signing one file, that seems like a fine idea to me too.